src/cleanup_translate.py

```diff
@@ -0,0 +1,26 @@
+import os
```

Comment:
Can you comment this please? Even high level just high level what each for loop is doing. also can you give an example of the regex that you are searching for. also, let's put this in a function main() and call main() at the bottom of the script.

Reply:
Also, wouldn't it be better to come up with a list of keys that aren't used? so you would get the list of keys that are used, look at the json files and see all the keys that are there, and then subtract the keys that are used to get the ones we don't use. if that makes sense. cause right no translate_keys.json contains a ton of keys because we use lots of them, but it doesn't highlight which ones we DONT use and those are the ones we acc care about and what to remove